src/components/Home.jsx

```diff
@@ -49,6 +49,9 @@ const Home = ({ currentLanguage, dispatch, history, rtl, translate }) => {
         <div>
           <ZooniverseLogotype width="100px" />
           <h1>{translate('topNav.site')}</h1>
+          {language !== LANGUAGES.ENGLISH && (
+            <h1>Scribes of the Cairo Geniza</h1>
```

Comment:
I'm not sure what's happening here. Does this add a second `h1` if the language isn't English? Documents should really only have one top-level heading.

Reply:
The research team wanted the English variant kept when switching to Arabic and Hebrew. I can change it to `h2`, as that would probably make more sense.